code/datums/traits/_quirk.dm

```diff
@@ -29,7 +29,16 @@
 	add()
 	if(spawn_effects)
 		on_spawn()
-		addtimer(CALLBACK(src, .proc/post_add), 30)
+	if(quirk_holder.client)
+		post_add()
+	else
+		RegisterSignal(quirk_holder, COMSIG_MOB_LOGIN, .proc/on_quirk_holder_first_login)
+
+
+/// Run post spawn text and client preferences
```

Comment:
```suggestion
///Run post spawn text and client preferences.
```
No space between the 3 `/` and the comment, else it won't be properly documented.

Reply:
really? that is a thing? *waves bye to the space*

Reply:
Block comments for style.